### ISSUE TITLE
Improve Contact page

### ISSUE THREAD

**Issue by MyfanwyNixon** [mysociety/writetothem#369]
https://www.writetothem.com/about-contact

![fireshot screen capture 1013 - writetothem - contact technical support - www_writetothem_com_about-contact](https://cloud.githubusercontent.com/assets/944656/23016135/65512f10-f42d-11e6-800a-aeffc37cd1cb.png)

If a user selects 'your representative', shouldn't a big error message come up to tell them to go back to the homepage and put in their postcode?

Emails intended for MPs are still one of the most frequent user support tasks. In a Slack chat yesterday, it was suggested that we could equate the amount of time spent on replying to these users to the amount of time we might dedicate to fixing the problem, but I don't think that's quite right: 
a) If we don't fix it, the problem is ongoing, forever
b) users won't always return to WTT and start again when we write back and tell them to, and anyway we've introduced a gap of up to 72 hours (if it happens to span a weekend) between what they intended to do and our telling them they got it wrong
c) It leaves a negative feeling of having done something stupid with the user, and this will rub off on their feelings about the site

ref #21

**Comment by abimysoc** [comment]
That's already what happens, but only after you submit the contact form with the 'your representative' option selected.

We could seamlessly shove them into the writing process from the contact form _as soon as they pick that radio button_: show 'em a box that says "Enter your postcode here to get started".

**Comment by MyfanwyNixon** [comment]
> That's already what happens, but only after you submit the contact form with the 'your representative' option selected.

Aha, that explains it - I hadn't got that far (obviously!)

**Comment by MyfanwyNixon** [comment]
So wait, these people are composing their letter, submitting, getting the error message, then returning to the form and selecting an option other than 'your representative' all in order to send their message to the wrong place?

**Comment by abimysoc** [comment]
Not all the messages intended for reps come via the form, but yes. There's too much text on the contact page and they're not reading it. Possibly they _can't_ read or understand it, given the low literacy levels in some of the misdirected messages. We could help them better with design.

See also #171, perhaps - too hard for people to figure out who to write to / do we hide MPs too far down the list?

**Comment by MyfanwyNixon** [comment]
Agree. And yes, looking at text like "Contact your Councillors, MP, MEPs, MSPs, or NI, Welsh and London Assembly Members" I'm guessing that's just confusing for someone not terribly au fait with the UK political system.

**Comment by MyfanwyNixon** [comment]
Changing title of this ticket and bumping it up to the top.
In Slack today we speculated that:
- there's too much text on the page
- people who contact us via this form meaning to contact their representatives often have apparently low levels of literacy
- "Contact your Councillors, MP, MEPs, MSPs, or NI, Welsh and London Assembly Members" may be the most prominent message here, but not in the way we want it to be (ie people think 'oh good, I'm on the right page')
- we should check with Abi how the improvements to the mysociety.org contact page have (or have not) improved things, and which features might usefully be adopted here as well.

**Comment by abimysoc** [comment]
As I said in February: https://github.com/mysociety/orgsites/issues/988#issuecomment-367044095 (and I have no reason to revise my view since then).

**Comment by MyfanwyNixon** [comment]
Just to add a note that this happened again today and the correspondent was definitely not of a low literacy level. The email contained highly sensitive material which is an impetus in itself to get this fixed.

**Comment by zarino** [comment]
A quick analysis of the last 3 months of emails via the WTT contact form showed the following breakdown:

<table>
<tr><th colspan="2">Emails we want</th></tr>
<tr><td>33</td><td>missing / incorrect contact details for representative</td></tr>
<tr><td>1</td><td>email to representative bounced</td></tr>
<tr><td>2</td><td>giving feedback on wording of our follow-up questionnaire</td></tr>
<tr><td>12</td><td>question about site functionality (eg: stats, integrations, edge-cases)</td></tr>
<tr><td>2</td><td>reporting technical error or bug</td></tr>
<tr><td>1</td><td>feedback about theyworkforyou.com</td></tr>
<tr><th colspan="2">Emails we don’t mind</th></tr>
<tr><td>3</td><td>missing confirmation email</td></tr>
<tr><td>1</td><td>want to edit an email they sent via the service</td></tr>
<tr><td>1</td><td>want a copy of an email they sent via the service</td></tr>
<tr><td>5</td><td>letting us know that their representative replied</td></tr>
<tr><td>2</td><td>asking for direct access to representative contact details</td></tr>
<tr><td>2</td><td>asking to unsubscribe</td></tr>
<tr><th colspan="2">Emails we don’t want</th></tr>
<tr><td>3</td><td>mistaken us for representative (benefits / housing / tenancy)</td></tr>
<tr><td>2</td><td>mistaken us for representative (neighbourhood dispute)</td></tr>
<tr><td>4</td><td>mistaken us for representative (other)</td></tr>
<tr><td>3</td><td>"I’ve not heard back from my MP, what should I do?"</td></tr>
<tr><td>5</td><td>"I’ve received an unsatisfactory response, how do I escalate this?"</td></tr>
<tr><td>1</td><td>"Help me write to my MP"</td></tr>
<tr><td>4</td><td>spam / unintelligible</td></tr>
</table>

Based on that, it feels like there are 3 prime candidates for special treatment, to help divert people away from contacting us:

1. Messages meant for their representative (or government more generally!)
2. Questions about unresponsive MPs
3. Questions about escalation after MPs are unhelpful

If we can double down on preventing those messages, we’d take our "don’t want" emails down from around 25% of the total support mailbag, to nearer 5%.

So I’m thinking we answer those three things _first_ before we let them send a message. And rather than telling them what they _can’t_ use the page for, we tell them what they _can_ do, similar to the recent mysociety.org/contact redesign.

Anyway, here’s a mockup – what do you think @abimysoc @MyfanwyNixon @wrightmartin? Still too wordy?

![contact-us](https://user-images.githubusercontent.com/739624/46799213-1cd91f00-cd4c-11e8-9842-e0d37b0f6b55.png)

**Comment by MyfanwyNixon** [comment]
I have to take issue with "letting us know that their representative replied" being under 'emails we don't mind' - this is either:
- people replying to our questionnaire instead of clicking one of the links (which is a different issue, as those mails don't come through the 'contact' page ; or
- people mistaking us for a campaign that sent them our way.

**Comment by MyfanwyNixon** [comment]
Summary of the conversation on Slack:
- Could we be more like Amazon/WDTK with simplified options, without very much text - see https://www.whatdotheyknow.com/help/contact or Abi's screenshot of 
![amazon](https://user-images.githubusercontent.com/944656/46804868-a3e1c380-cd5b-11e8-8144-291162bcf781.png)

I suspect it may be asking for trouble still having the contact form as the only text-accepting element on the page: people will scroll down to it without reading any of the text.
Other thoughts:
- WTT's muted theme works against itself here - is it worth breaking away from it to do something that's brighter coloured and more in your face?

- We can shorten the final header, since I believe that the vast majority of reps' email addresses come either in reply to messages sent to Democratic Services or via a different form, like this: https://www.writetothem.com/corrections?id=11454 So: report a problem or ask a technical question about this website'.

- Instead of 'we are not your MP' how about 'your message will not go to your MP'? 

- Given that we often detect a low level of literacy in the messages that come to us instead of MPs, is 'your local representative' too difficult? People may see that and think, 'no I want to write to my MP'. 
I'd be tempted to put 'Your MP or councillor' and 'or other representatives' in smaller lettering below. And instead of 'is your representative not responding', try 'you haven't received a reply'. Although, that said, again I think many of these come in reply to our 'did you get a response' survey - this might not need to be an option.

- Agree we should take the explanatory text away from options 2, 3 and 4.

**Comment by zarino** [comment]
Huge thanks to @abimysoc @MyfanwyNixon @wrightmartin for feedback in Slack. I shared a working prototype of a second iteration in the channel, but here are some stills:

![contact-us-2a](https://user-images.githubusercontent.com/739624/46812795-625a1400-cd6d-11e8-9a03-131ca83b3f73.png)

![contact-us-2b](https://user-images.githubusercontent.com/739624/46812800-65550480-cd6d-11e8-97ea-109d03d327f1.png)

![contact-us-2d](https://user-images.githubusercontent.com/739624/46812804-671ec800-cd6d-11e8-9f74-b2a9918286fd.png)

![contact-us-2e](https://user-images.githubusercontent.com/739624/46812807-68e88b80-cd6d-11e8-85bc-a767fcddb8b3.png)

**Comment by zarino** [comment]
Pull request here: https://github.com/mysociety/writetothem/pull/467

**Comment by MyfanwyNixon** [comment]
Abi suggests that we can place the user directly into the correct workflow (MP or councillor/s), but presumably only if we ask for their postcode. Which I guess we can do?
Do we want to be super clever-clever and not even acknowledge they've made a mistake, just take them to the next step of the process of contacting their MP/councillor? Or in the name of transparency do we explain what we're doing?

### **I've been treated unfairly by a government agency**
This is something to contact your MP about (**not us - we just run this website!**). Please input your postcode [on the homepage](https://www.writetothem.com/), then click on your MP's name to send them a message.

(Or! Postcode input field and straight to the compose pane)

### **I'm having a problem locally**
Your Local Councillor can help with this kind of issue (**not us - we just run this website!**). Please input your postcode [on the homepage](https://www.writetothem.com/), then click on your councillor's name to send them a message.

(Or! Postcode input field and straight to the compose pane - more complicated because they have to pick one or all of their councillors)

**Comment by abimysoc** [comment]
Revisiting after a month to report on success. Since 17th March (inclusive), we have had 116 messages via the contact form.

89 were spam, unfortunately (see #472).

27 were genuine messages from users, of which:
12 provided representatives' contact details
3 provided feature suggestions or other information they thought was useful/interesting
12 had an assortment of queries about or problems with using the website
and 0, none, zero were correspondence intended for MPs, councillors, gov departments, etc!
